03-Mar-2023 13:31:13.796 INFO [main] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 13:31:14.957 INFO [main] org.apache.catalina.core.ApplicationContext.log loginFilter:Initializing filter
03-Mar-2023 16:30:03.560 INFO [main] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 16:30:04.774 INFO [main] org.apache.catalina.core.ApplicationContext.log loginFilter:Initializing filter
03-Mar-2023 16:41:06.822 SEVERE [http-nio-9999-exec-16] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.822 SEVERE [http-nio-9999-exec-35] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.822 SEVERE [http-nio-9999-exec-32] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.822 SEVERE [48] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.823 SEVERE [http-nio-9999-exec-16] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.823 SEVERE [http-nio-9999-exec-35] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.823 SEVERE [48] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:41:06.823 SEVERE [http-nio-9999-exec-32] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:58:53.701 SEVERE [http-nio-9999-exec-43] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:58:53.701 SEVERE [http-nio-9999-exec-45] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:58:53.702 SEVERE [http-nio-9999-exec-47] org.apache.catalina.core.ApplicationContext.log Error during mapping
	java.lang.NullPointerException
		at org.apache.catalina.mapper.Mapper.internalMapWrapper(Mapper.java:868)
		at org.apache.catalina.mapper.Mapper.map(Mapper.java:725)
		at org.apache.catalina.core.ApplicationContext.getRequestDispatcher(ApplicationContext.java:457)
		at org.apache.catalina.core.ApplicationContextFacade.getRequestDispatcher(ApplicationContextFacade.java:225)
		at org.apache.catalina.connector.Request.getRequestDispatcher(Request.java:1440)
		at org.apache.catalina.connector.RequestFacade.getRequestDispatcher(RequestFacade.java:614)
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:58:53.702 SEVERE [http-nio-9999-exec-45] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:58:53.702 SEVERE [http-nio-9999-exec-43] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 16:58:53.702 SEVERE [http-nio-9999-exec-47] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [ListJobs] in context with path [/CRUD] threw exception
	java.lang.NullPointerException
		at controller.ListJobs.doGet(ListJobs.java:62)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 17:07:22.748 INFO [http-nio-9999-exec-48] org.apache.catalina.core.ApplicationContext.log CreateNewAccountServlet: CreateAccountServlet _Nameing:Cannot create PoolableConnectionFactory (Login failed for user 'sa'. ClientConnectionId:3f328c79-62b9-43ee-803b-7ca6c36de814)
03-Mar-2023 17:07:59.282 INFO [http-nio-9999-exec-53] org.apache.catalina.core.ApplicationContext.log CreateNewAccountServlet: CreateAccountServlet _Nameing:Cannot create PoolableConnectionFactory (Login failed for user 'sa'. ClientConnectionId:fba4d595-9b9d-4382-9810-442445d9c393)
03-Mar-2023 19:51:29.249 INFO [main] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 19:51:31.075 INFO [main] org.apache.catalina.core.ApplicationContext.log loginFilter:Initializing filter
03-Mar-2023 22:28:50.337 SEVERE [http-nio-9999-exec-45] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Home] in context with path [/Assignment] threw exception
	java.lang.NullPointerException
		at controller.Home.processRequest(Home.java:44)
		at controller.Home.doGet(Home.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 22:28:50.844 SEVERE [http-nio-9999-exec-46] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Home] in context with path [/Assignment] threw exception
	java.lang.NullPointerException
		at controller.Home.processRequest(Home.java:44)
		at controller.Home.doGet(Home.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 22:28:50.993 SEVERE [http-nio-9999-exec-31] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Home] in context with path [/Assignment] threw exception
	java.lang.NullPointerException
		at controller.Home.processRequest(Home.java:44)
		at controller.Home.doGet(Home.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:683)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:792)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:356)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1762)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
03-Mar-2023 23:41:34.566 INFO [http-nio-9999-exec-162] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:41:35.292 INFO [http-nio-9999-exec-160] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:41:40.345 INFO [Catalina-utility-2] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:43:13.684 INFO [http-nio-9999-exec-161] org.apache.catalina.core.ApplicationContext.log AdminFilter:doFilter()
03-Mar-2023 23:43:13.684 INFO [http-nio-9999-exec-161] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoBeforeProcessing
03-Mar-2023 23:43:13.849 INFO [http-nio-9999-exec-161] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoAfterProcessing
03-Mar-2023 23:43:13.852 INFO [http-nio-9999-exec-159] org.apache.catalina.core.ApplicationContext.log AdminFilter:doFilter()
03-Mar-2023 23:43:13.852 INFO [http-nio-9999-exec-159] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoBeforeProcessing
03-Mar-2023 23:43:13.852 INFO [http-nio-9999-exec-159] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoAfterProcessing
03-Mar-2023 23:43:16.286 INFO [http-nio-9999-exec-162] org.apache.catalina.core.ApplicationContext.log AdminFilter:doFilter()
03-Mar-2023 23:43:16.286 INFO [http-nio-9999-exec-162] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoBeforeProcessing
03-Mar-2023 23:43:16.293 INFO [http-nio-9999-exec-162] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoAfterProcessing
03-Mar-2023 23:43:17.485 INFO [http-nio-9999-exec-165] org.apache.catalina.core.ApplicationContext.log AdminFilter:doFilter()
03-Mar-2023 23:43:17.485 INFO [http-nio-9999-exec-165] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoBeforeProcessing
03-Mar-2023 23:43:17.528 INFO [http-nio-9999-exec-165] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoAfterProcessing
03-Mar-2023 23:43:18.132 INFO [http-nio-9999-exec-167] org.apache.catalina.core.ApplicationContext.log AdminFilter:doFilter()
03-Mar-2023 23:43:18.132 INFO [http-nio-9999-exec-167] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoBeforeProcessing
03-Mar-2023 23:43:18.135 INFO [http-nio-9999-exec-167] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoAfterProcessing
03-Mar-2023 23:43:22.752 INFO [http-nio-9999-exec-157] org.apache.catalina.core.ApplicationContext.log AdminFilter:doFilter()
03-Mar-2023 23:43:22.752 INFO [http-nio-9999-exec-157] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoBeforeProcessing
03-Mar-2023 23:43:22.754 INFO [http-nio-9999-exec-157] org.apache.catalina.core.ApplicationContext.log AdminFilter:DoAfterProcessing
03-Mar-2023 23:46:29.536 INFO [http-nio-9999-exec-172] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:50:43.838 INFO [http-nio-9999-exec-180] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:51:08.548 INFO [http-nio-9999-exec-183] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:51:37.196 INFO [http-nio-9999-exec-178] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:56:12.642 INFO [http-nio-9999-exec-175] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:57:44.995 INFO [http-nio-9999-exec-198] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:57:49.585 INFO [http-nio-9999-exec-194] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:57:52.332 INFO [Catalina-utility-2] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:59:06.970 INFO [http-nio-9999-exec-205] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
03-Mar-2023 23:59:12.893 INFO [Catalina-utility-1] org.apache.catalina.core.ApplicationContext.log AdminFilter:Initializing filter
